conda_requirements.txt

```diff
@@ -7,3 +7,5 @@ pandas>=0.18.1
 scikit-learn==0.18.1
 seaborn==0.7.1
 ipython==5.1.0
+python-wget=3.2 
```

Comment:
Ooops… my bad. I shouldn't have tried it with `pip` ; )

Just out of curiosity, why not using native [`urlretrieve`](https://docs.python.org/3.6/library/urllib.request.html#urllib.request.urlretrieve) that already has a straightforward API?